src/Compilers/CSharp/Test/Semantic/Semantics/InterceptorsTests.cs

```diff
@@ -7199,4 +7199,211 @@ static void Main()
         Assert.Equal("(7,9)", locationSpecifier.GetDisplayLocation());
         AssertEx.Equal("""[global::System.Runtime.CompilerServices.InterceptsLocationAttribute(1, "jB4qgCy292LkEGCwmD+R6FIAAAA=")]""", locationSpecifier.GetInterceptsLocationAttributeSyntax());
     }
+
+    [Fact]
+    public void ConditionalAccess_01()
+    {
+        var source = CSharpTestSource.Parse("""
+            class C
+            {
+                void M() => throw null!;
+
+                static void Main()
+                {
+                    var c = new C();
+                    c?.M();
+                }
+            }
+            """, "Program.cs", RegularWithInterceptors);
+
+        var comp = CreateCompilation(source);
+        var model = comp.GetSemanticModel(source);
+        var node = source.GetRoot().DescendantNodes().OfType<InvocationExpressionSyntax>().Single();
+        var locationSpecifier = model.GetInterceptableLocation(node)!;
+
+        var interceptors = CSharpTestSource.Parse($$"""
+            #nullable enable
+            using System;
+
+            static class Interceptors
+            {
+                {{locationSpecifier.GetInterceptsLocationAttributeSyntax()}}
+                public static void M1(this C? c) => Console.Write(1);
```

Comment:
>C? [](http://example.com/codeflow?start=43&length=2)

Do we need the `?` annotation on `C` here? That gives `M1` a different signature than `M`.
nit: consider also verifying that `M` receives the proper values and is not invoked in the null case.